athena/modules/text/module_text_llm/module_text_llm/basic_approach/generate_suggestions.py

```
@@ -1,97 +1,124 @@
 from typing import List
-
 from athena import emit_meta
 from athena.text import Exercise, Submission, Feedback
 from athena.logger import logger
 from llm_core.utils.llm_utils import (
-    get_chat_prompt_with_formatting_instructions, 
-    check_prompt_length_and_omit_features_if_necessary, 
+    get_chat_prompt,
+    check_prompt_length_and_omit_features_if_necessary,
     num_tokens_from_prompt,
 )
```

Pylint (3.3.7)

[error] Unable to import 'llm_core.utils.llm_utils'

(E0401)